gamestonk_terminal/forex/fx_controller.py

```diff
@@ -152,6 +185,43 @@ def call_calendar(self, other_args: List[str]):
         """Call calendar"""
         fx_view.calendar(self.instrument, other_args)
 
+    def call_news(self, other_args: List[str]):
+        """Call news [News API]"""
+        news_view.news(other_args, self.instrument)
+
+    def call_bullbear(self, other_args):
+        """Call bullbear fom stocktwits"""
+        instrument = fx_view.format_instrument(self.instrument, " ")
+        stocktwits_view.bullbear(other_args, instrument)
+
+    def call_messages(self, other_args):
+        """Call messages from stocktwits"""
+        instrument = fx_view.format_instrument(self.instrument, " ")
+        stocktwits_view.messages(other_args, instrument)
+
+    def call_edasummary(self, other_args: List[str]):
```

Comment:
Instead of having 1 command for each of the EDA commands.

What do you think about having a menu that would dive into EDA?

The sequence of actions would be:
* You add the `> eda` sub-menu
* Then when user does `eda` you can do the following:
```
def call_eda(self, _):
        """Call the EDA menu"""
        eda_controller.menu(df_stock, ticker, s_start, s_interval)
```
* The only thing you need to take care of is the variables:
    * df_stock
    * ticker
    * s_start
    * s_interval
But this way you'd get the full menu in 1 simple call!

This is just an idea! Let me know what you think!

Reply:
This is a great idea! I didn't think to do that.